packages/vkui/src/components/ActionSheet/ActionSheet.tsx

```diff
@@ -17,12 +18,12 @@ import { ActionSheetDropdownSheet } from './ActionSheetDropdownSheet';
 import type { SharedDropdownProps } from './types';
 import styles from './ActionSheet.module.css';
 
-type CloseInitiators = 'action-item' | 'cancel-item' | 'other';
+type CloseReason = 'action-item' | 'cancel-item' | 'other';
 export interface ActionSheetOnCloseOptions {
   /**
    * Причина закрытия всплывающего элемента.
    */
```

Comment:
```suggestion
/**
  * @deprecated Since 8.0.0 аргумент, переданный в функцию `onClosed`, устарел и будет удален в **VKUI v10**.
   * Для получения причины закрытия всплывающего окна используйте свойство `onClose`.
  */
export interface ActionSheetOnCloseOptions {
/**
  * Причина закрытия всплывающего элемента.
  *
  * @deprecated Since 8.0.0 аргумент, переданный в функцию `onClosed`, устарел и будет удален в **VKUI v10**.
  * Для получения причины закрытия всплывающего окна используйте свойство `onClose`.
  */
```

тогда в IDE будет видно, что депрекейтит

<img width="226" height="78" alt="Image" src="https://github.com/user-attachments/assets/4a04abb6-8b23-4219-b51d-de265ad5b861" />

ну и те, кто `ActionSheetOnCloseOptions` используют увидет, что депрекейт